lib/vnet/service_windows.go

```diff
@@ -146,9 +168,16 @@ loop:
 				status <- svc.Status{State: state, Accepts: cmdsAccepted}
 			case svc.Stop:
 				slog.InfoContext(ctx, "Received stop command, shutting down service")
+				// Cancel the context passed to s.run to terminate the
+				// networking stack.
 				cancel()
+				terminateTimedOut = cmp.Or(terminateTimedOut, time.After(terminateTimeout))
 				status <- svc.Status{State: svc.StopPending}
 			}
+		case <-terminateTimedOut:
+			slog.ErrorContext(ctx, "Networking stack failed to terminate within %v, exiting process")
```

Comment:
Missing arg for `%v`.

Reply:
oh thanks and this is slog too where i shouldn't even be using %v